samples/k8s-connectedcluster-only/README.md

```diff
@@ -34,9 +34,9 @@ Following is the guidance on the supportedClusterTypes:
 | Property      | Description |
 | -----------   | ----------- |
 | supportedClusterTypes       | Contains an object for each top-level cluster-type. Allowed types are – “managedClusters”, “connectedClusters”. "managedClusters" denotes Azure Kubernetes Service (AKS) clusters. "connectedClusters" denotes Azure Arc-enabled Kubernetes clusters. For each of these cluster types, specify distributions and unsupported Kubernetes versions for these distributions. If supportedClusterTypes is not provided, all distributions of ‘managedClusters’ will be supported by default. If supportedClusterTypes is provided, and a given top level cluster type is not provided, then all distributions and Kubernetes versions for that cluster type will be treated as unsupported. |
```

Comment:
For Azure Kubernetes Service clusters, can you point to link https://learn.microsoft.com/en-us/azure/aks/what-is-aks
For Azure Arc-enabled Kubernetes clusters, can you point to link https://learn.microsoft.com/en-us/azure/azure-arc/kubernetes/overview